runtime/POSIX/klee_init_env.c

```diff
@@ -60,16 +60,18 @@ static int __streq(const char *a, const char *b) {
   return 0;
 }
 
-static char *__get_sym_str(int numChars, char *name) {
+static char *__get_sym_str(int maxSize, char *name) {
   int i;
-  char *s = malloc(numChars+1);
+  int argSize = klee_range(0, maxSize + 1, "argSize");
+
+  char *s = malloc(argSize + 1);
```

Comment:
What if s==NULL?

Reply:
The allocation in KLEE never fails, so this should be ok (as long as it is linked to bitcode by KLEE and not by Symbiotic).